infra/src/main/kotlin/fi/fta/geoviite/infra/linking/SwitchLinkingService.kt

```diff
@@ -1245,10 +1246,41 @@ class SwitchLinkingService @Autowired constructor(
             message = "Cannot link a plan that is hidden", localizedMessageKey = "plan-hidden"
         )
     }
+
+    fun createSwitchLinkingParameters(
+        suggestedSwitch: SuggestedSwitch,
+        layoutSwitchId: IntId<TrackLayoutSwitch> = temporarySwitchId,
```

Comment:
Ei suorilta tämän reviewin asiaa, mutta tämän vakion VOISI_NIMETÄ_NÄIN

Reply:
Tuo tulee itse asiassa siitä että kotlin ei anna tehdä const-vakioita olioista, vain primitiiveistä. Vähän makuasia että pitäisikö ne silti nimetä vakio-tyyliin. Taitaa tällä hetkellä olla molempia tapoja käytössä. Ehkä nuo voisi olla all-caps.

Reply:
Also, tuo temporarySwitchId nostaa kyllä vähän kulmakarvoja. Miksikäs sellainen vakio on olemassa? Jos tyypille on mahdollista että id puuttuu niin eikös sen pitäisi vain olla nullable? Tuollainen vakio-id on aika suspect.

Reply:
Tuo temporarySwitchId on peräisin vaihdelinkitysehdotuksen laskennan muutoksesta muistinvaraiseksi. 

Vaihdelinkitysehdotuksen luonnin aikana ei aiemmin linkittämättömälle vaihteelle ole olemassa vaihdetunnusta (eikä se sinänsä sellaista kaipaakkaan), mutta jokin vaihdetunnus tarvitaan linkityslogiikan tietorakenteissa (en muista tarkalleen mille kaikille, mutta ainakin topologialinkit määritellään vaihdetunnus+vaihdepistetunnus -pareina).

Voisi tuon varmaan toteuttaa muutenkin, etenkin nyt kun linkitys itsessään muuttui/muuttuu muistinvaraiseksi.